WowUp.WPF/Utilities/StartupOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CommandLine;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WowUp.Common.Enums;
+
+namespace WowUp.WPF.Utilities
+{
+    public class StartupOptions
+    {
+        [Option(shortName: 'i', longName: "install", HelpText = "Specify addon urls to install them")]
+        public IEnumerable<string> InputURLs { get; set; }
+        [Option(shortName: 's', longName: "silent", HelpText = "Start application minimized")]
+        public bool Silent { get; set; }
+        [Option(shortName: 'u', longName: "update", HelpText = "Update non-ignored addons on startup")]
```

Comment:
An update option is probably not necessary, if they start the app minimized the session service will run auto update automatically.

Reply:
Well, this is not about auto-updates. The idea here is to update all addons that can be updated (except ignored). Like, emulate the Update All button click.

Reply:
That seems to go against the entire concept of marking things auto-update.

Reply:
You have a point. Removed -u and -f. Let auto-updates do everything.